### DISCUSSION TITLE
freeplane menu are grey and deactivated

### DISCUSSION THREAD

**Discussion by dfedra** [freeplane/freeplane#1957] [Help]
I'm a newbee using freeplane and just created a first small mindmap but when I re-opened it, i am no longer able to edit my entries nor to add or delete any node...? I'm not aware of any changes only that I stored my mindmap in another directory than usually used by freeplane.
The status line shows "Mindmap mode = show"  / Mindmap-(Modus Anzeigen)
![Bildschirmfoto vom 2024-08-09 12-19-59](https://github.com/user-attachments/assets/72b6b6ce-eeee-47ea-8541-c048faf0633d)
How can I change / infuence this mode?

I'm using freeplane version 1.10.5 on a Linux Ubuntu 22.04

**Comment by macmarrum** [comment]
In Freeplane, you can switch between "Editing" and "Viewing" modes.

Try menu `Bearbeiten->Bearbeiten`

There are a lot of commands available in Freeplane, so when I'm lost I try to use `Help->Command Search` | `Hilfe->Befehlssuche`

**Comment by dfedra** [comment]
Thank you so much, this was exactly the point, for some  reason i obviously unintentionally did this (wrong) click and was not aware of it...
-------- Ursprüngliche Nachricht --------Von: macmarrum ***@***.***> Datum: 09.08.24  19:18  (GMT+01:00) An: freeplane/freeplane ***@***.***> Cc: dfedra ***@***.***>, Author ***@***.***> Betreff: Re: [freeplane/freeplane] freeplane menu are grey and deactivated (Discussion #1957) 
In Freeplane, you can switch between "Editing" and "Viewing" modes.
Try menu Bearbeiten->Bearbeiten
There are a lot of commands available in Freeplane, so when I'm lost I try to use Help->Command Search | Hilfe->Befehlssuche
